### DISCUSSION TITLE
Reason for separate URL?

### DISCUSSION THREAD

**Discussion by fishcharlie** [community/community#9761] [Other Feature Feedback, Questions, & Ideas] [answered]
Is there a reason that Code Search is on a different URL and not integrated directly into the GitHub.com interface?

**Comment by thinkh** [comment, marked as answer]
Probably because the Code Search is still a Technology Preview. I guess when it becomes generally available it will be integrated into GitHub.com.

**Reply by look** [reply]
That's basically correct. We address this [in the FAQ](https://cs.github.com/about/faq#new-ui):

> Why is this separate from the github.com user interface?
>
> It’s separate only so that we can quickly iterate on the UI/UX and feature set without breaking github.com for millions of people. When it’s ready, we plan to integrate these improvements back into the main github.com interface for searching, browsing, and navigating code.
